### ISSUE TITLE
reCaptcha never validating on mobile device

### ISSUE THREAD

**Issue by natdean** [google/recaptcha#245]
I have a reCaptcha on an email share modal. It just recently stopped working on mobile. The recaptcha checkbox will spin continuously until you reload the page. Have there been any recent changes that might cause this bug? I cannot figure it out!

**Comment by rowan-m** [comment]
I've just tried all the examples on https://recaptcha-demo.appspot.com and they're all working. Do you have an example where this is not working?

**Comment by natdean** [comment]
Sure, If you go to this page on a mobile device: https://www.modaoperandi.com/alexis-pf18/edaline-cropped-jumpsuit and click on "Drop A Hint", then click the recaptcha you will see that it gets stuck and does not allow you to exit the screen unless you reload the whole page. The expected behavior is that it submits the form and returns the user to the PDP. 
This only occurs on mobile devices and works great on desktop still!

**Comment by rowan-m** [comment]
What browser are you using? I just tried Chrome Beta, Samsung Internet, and Opera Mina from a Pixel 1 and was able to complete the challenge.
![screenshot_20180803-154849](https://user-images.githubusercontent.com/108052/43649663-a3bb1c28-9735-11e8-9dbd-d63050b8481b.png)

**Comment by natdean** [comment]
Hmm well I'm not sure what happened but now it is working again on all mobile devices and browsers. I was seeing it on iphone safari, chrome, and firefox. Thanks for looking into it anyways @rowan-m. 
@Moldjuice do you see the issue anymore?

**Comment by rstainbrookUW** [comment]
I also saw this issue yesterday.  I experienced issues in IE11 and Edge on Windows 10 and Safari and Chrome on iOS 11.  I tried on our site but also others, including https://www.google.com/recaptcha/api2/demo.  Both FireFox and Chrome worked fine on Windows 10 for all reCaptcha I tried.  I noticed the issue at 3pm central time and it seemed to clear up around 4:30pm central time.

**Comment by Moldjuice** [comment]
@natdean, it seems functional again on for me too. If someone has an explanation (to be reassured), I am interested because these forms are in prod. 
Have a good day everyone

**Comment by rowan-m** [comment]
It's possible there may have been network or service issues within that time, bit that's outside the scope of this repo.

**Comment by sgleisner** [comment]
Just for the record, I faced the same problem and it turned out to be the zoom.

Steps to replicate:
1. Open a site that has a reCAPTCHA (e.g. https://recaptcha-demo.appspot.com/recaptcha-v2-checkbox.php)
2. In the the device toolbar, select a mobile device (e.g. Galaxy S5)
3. In the same toolbar, set the zoom level to a zoom different than 100% (e.g. 75%)
4. Hover the mouse over the captcha zone and click it.

Expected behavior:
When the pointer enters the captcha zone, it should change from the "touch" circle to the OS' mouse pointer.
Once inside the captcha zone, when clicking it, its checkbox should load and then show a green check mark.

Actual behaviour:
The pointer does not change when entering the captcha zone and nothing happens when clicking.
If the zoom level is set up back to 100% and the site refreshed, it works as expected.

Environment:
Ubuntu 18.04 64bit
Unfortunately, I just updated chrome to see if the bug if fixed, and it is fixed indeed on version 78.0.3904.108, but I cannot recall what version I had when I had this problem
